dogpile/cache/backends/redis.py

```diff
@@ -292,6 +302,8 @@ class RedisSentinelBackend(RedisBackend):
      asynchronous runners, as they run in a different thread than the one
      used to create the lock.
 
+     .. versionadded:: 1.3,2  Added ``username`` parameter.
```

Comment:
write it like this:

    .. versionadded:: 1.3.1  Added ``username`` parameter.

Reply:
Fixed in the latest version.